components/CardContent.js

```diff
@@ -10,10 +10,14 @@ const propTypes = {
   cardLink: PropTypes.node,
   cardInfo: PropTypes.array,
   className: PropTypes.string,
+  iconDescription: PropTypes.string,
 };
 
 const defaultProps = {
+  iconDescription: 'card icon',
   cardIcon: 'app-services',
+  cardTitle: 'card title',
+  cardInfo: ['info']
```

Comment:
Hmm, should this one be a default prop? Looking at line 44 it's currently a conditional whether to render this element or not